### ISSUE TITLE
Maybe move into single docker-compose.yml file? 

### ISSUE THREAD

**Issue by alexmacarthur** [alexmacarthur/local-docker-db#2]
Instead of having each DB recipe split out into its own directory, we could have everything in a single `docker-compose.yml` file. Are there significant advantages to doing it one way over the other?

**Comment by Abhinav1217** [comment]
I actually just used your mongo db compose file. My exposer to docker is just as-needed bases since I am simply a php web developer. Having separate file is both educational and someone can simply "curl" it from github for new project. 

In fact I have actually added the [raw githubusercontent](https://raw.githubusercontent.com/alexmacarthur/local-docker-db/master/mongo/docker-compose.yml) url in my shell script and ran that shell script to scaffold my new project. The script simply "wget" the docker-compose file in the approporiate folder and I just had to adjust the `volumes` line to get started with my work.

Thanks for the script by the way. It was like google knew I was going to need it today and gave this link on my morning feeds. Kinda scary to think about.
